standalone/ansible/postgres.yml

```diff
@@ -2,3 +2,4 @@
 - hosts: postgres
   roles:
     - { role: postgres, tags: ['postgres'] }
+    - { role: postgres_replication, tags: ['postgres_replication'] }
```

Comment:
So this line ensures that the server running the primary postgres will set up its replication requirements. I suspect that means it will run the contents of the `master.yml` file below.

Where does the replica live? Does it live on another server, and if so, where are we asking that server to run the corresponding `replica.yml`?

Reply:
This line runs the role on the postgres group (which contains both `postgres_primary` and `postgres_secondary`). It runs `postgres_replication/main.yml` which delegates the right tasks based on the `replication_role` like so:
https://github.com/simpledotorg/deployment/blob/6cd309452cec50659e78dc23e427684d980331c0/standalone/ansible/roles/postgres_replication/tasks/main.yml#L2

The `replication_role` is being set in the `hosts` file.